### DISCUSSION TITLE
Do not install component during upgrade

### DISCUSSION THREAD

**Discussion by kalyani1217** [wixtoolset/issues#6918] [Questions]
I have a component say web.config, in this component I have included a web.config file.

```
<Component Directory="FeatureA" Guid="{692AA53D-4DFB-4FA0-A6FD-D30905383D1D}" Id="web.config" Permanent="yes">

  <File KeyPath="yes" Source="Content\web.config" />

</Component>
```
During upgrades, I dont want to install this component. So, I have added a condition

```
<Component Directory="FeatureA" Guid="{692AA53D-4DFB-4FA0-A6FD-D30905383D1D}" Id="web.config" Permanent="yes">
    <Condition>[WixBundleInstalled]=0</Condition>
    <File KeyPath="yes" Source="Content\web.config" />

</Component>
```
The above condition not working during upgrades. Component is installing during upgrades. What should I add inside condition, so that the component should not install during upgrades?

**Comment by robmen** [comment]
`WixBundleInstall` is a Burn variable. It is not set in your MSI (unless you create a `Property` with that name, but that would be pretty confusing).
